src/pages/ProductsList/components/LeftSide/OptionChecked.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { fontMix } from '../../../../styles/mixin';
+import { useSearchParams } from 'react-router-dom';
+
+const OptionChecked = ({ item: { id, category, type, theme } }) => {
+  const [checked, setChecked] = useState(false);
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  useEffect(() => {
+    if (checked === true) {
```

Comment:
```suggestion
    if (checked) {
```
checked라는 값 자체가 boolean 값이기 때문에 다시 boolean값과 비교하지 않아도 됩니다